From 0ec1b3d20d612325b9c55baa2539d080eb6a72a8 Mon Sep 17 00:00:00 2001
From: Vishnu Banavath <vishnu.banavath@arm.com>
Date: Sat, 13 Nov 2021 08:34:42 +0000
Subject: [PATCH] se-proxy:dts: add se-proxy as child node

se-proxy sp string should be added for se-proxy node to be
read properly.

Signed-off-by: Vishnu Banavath <vishnu.banavath@arm.com>
Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Abdellatif El Khlifi <abdellatif.elkhlifi@arm.com>


---
 .../se-proxy/opteesp/default_se-proxy.dts.in  | 20 ++++++++++---------
 1 file changed, 11 insertions(+), 9 deletions(-)

diff --git a/deployments/se-proxy/opteesp/default_se-proxy.dts.in b/deployments/se-proxy/opteesp/default_se-proxy.dts.in
index 961071ad..9f5cf712 100644
--- a/deployments/se-proxy/opteesp/default_se-proxy.dts.in
+++ b/deployments/se-proxy/opteesp/default_se-proxy.dts.in
@@ -7,13 +7,15 @@
 @DTS_TAG@
 
 @DTS_NODE@ {
-	compatible = "arm,ffa-manifest-1.0";
-	ffa-version = <0x00010000>; /* 31:16 - Major, 15:0 - Minor */
-	uuid = <@EXPORT_SP_UUID_DT@>;
-	description = "SE Proxy";
-	execution-ctx-count = <1>;
-	exception-level = <1>; /* S-EL0 */
-	execution-state = <0>; /* AArch64 */
-	xlat-granule = <0>; /* 4KiB */
-	messaging-method = <0>; /* Direct messaging only */
+	se-proxy {
+		compatible = "arm,ffa-manifest-1.0";
+		ffa-version = <0x00010000>; /* 31:16 - Major, 15:0 - Minor */
+		uuid = <@EXPORT_SP_UUID_DT@>;
+		description = "SE Proxy";
+		execution-ctx-count = <1>;
+		exception-level = <1>; /* S-EL0 */
+		execution-state = <0>; /* AArch64 */
+		xlat-granule = <0>; /* 4KiB */
+		messaging-method = <0>; /* Direct messaging only */
+	};
 };
